recipes-omnect/initrdscripts/omnect-os-initramfs/fs-mount

```diff
@@ -30,7 +30,7 @@ run_fs_mount()
 
     # mount boot
     check_fs "boot"
-    mount /dev/omnect/boot "${ROOTFS_DIR}/boot" || return 1
+    mount -t vfat /dev/omnect/boot "${ROOTFS_DIR}/boot" || return 1
```

Comment:
why do we need that? what if we have a target with a non vfat boot partition?

Reply:
in grub context we have mount the boot partition with the type option vfat as well. could remove it here anyway.

Reply:
please do.

Reply:
done